### PR TITLE
Ratings updated

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: controllers/rating/rating.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
-const asyncHandler = require('../../middleware/async');
-const Product = require('../../models/Product');
-const ErrorResponse = require('../../utils/errorResponse');
+const asyncHandler = require("../../middleware/async");
```
Comment (oleksandrhodzevych) [inline controllers/rating/rating.js:1]: " to '

File: controllers/rating/rating.js [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
-    res.status(200).send('rating updated');
-})
-module.exports = { updateRating };
\ No newline at end of file
+  const { id } = req.params;
+  const { rate, userId } = req.body;
+
+  const productToUpdate = await Product.findById(id);
+  if (!productToUpdate) {
+    return next(new ErrorResponse("Product not found", 404));
```
Comment (oleksandrhodzevych) [inline controllers/rating/rating.js:11]: move to const all error messages

File: controllers/rating/rating.js (hunk 1 above)
Comment (bandvov) [reply controllers/rating/rating.js:1]: fixed

File: controllers/rating/rating.js (hunk 2 above)
Comment (bandvov) [reply controllers/rating/rating.js:11]: fixed

**Pull request closed without merging**